src/appleseed.cli/CMakeLists.txt

```diff
@@ -59,11 +59,11 @@ add_executable (appleseed.cli
     ${appleseed.cli_sources}
 )
 
-if (CMAKE_SYSTEM_NAME STREQUAL "Linux")
+if (UNIX AND NOT WIN32 AND NOT APPLE)
```

Comment:
OK, so what shall I pick: just `if (UNIX AND NOT APPLE)` or `if (UNIX AND NOT APPLE AND NOT CYGWIN)`?  Provided I know little about Cygwin, it can be wrong yet cluttering the code...